sql/cat_tools.sql

```diff
@@ -396,10 +402,14 @@ BEGIN
   END IF;
   RAISE DEBUG 'v_work "%"', v_work;
 
+  -- Get function name
+  function_name := r_trigger.tgfoid::regproc;
+
   -- Get function arguments
   v_execute_clause := ' EXECUTE PROCEDURE ' || r_trigger.tgfoid::regproc || E'\\(';
   v_array := regexp_split_to_array( v_work, v_execute_clause );
-  function_arguments := rtrim( v_array[2], ')' ); -- Yank trailing )
+  function_arguments :=  array_remove(regexp_split_to_array(rtrim( v_array[2], ')' ), '\W+'), '');
```

Comment:
I wanted to do this originally, but this will fail if you have any whitespace in any of the arguments themselves. I don't think it's really worth the (rather considerable) effort it would take to make this fool-proof.